using steamidfinder.com instead of steamid.io

SuperRayss opened the pull request:
steamid.io was updated to only show the alias used from the last time a profile was public. steamidfinder doesn't have this problem, and will show the current alias

SuperRayss committed:
using steamidfinder.com instead of steamid.io

harbingerofme commented:
I've said this in discord, but I'll repeat it here for clarity sake.

I'm looking into adding multiple name providers. Which is why this pr is not merged, but I will use this pr's work as a basis for those providers. Thank you very much.

SuperRayss committed:
- Fixed username parsing from web request
- Disabled features which were implemented to base client (UI sticking, console commands)
